src/client/Microsoft.Identity.Client/Internal/Logger/MsalLogger.cs

```diff
@@ -192,7 +192,19 @@ public void Log(LogLevel logLevel, string messageWithPii, string messageScrubbed
                 bool isLoggingPii = messageWithPiiExists && PiiLoggingEnabled;
                 string messageToLog = isLoggingPii ? messageWithPii : messageScrubbed;
 
-                string log = $"{isLoggingPii} MSAL {s_msalVersionLazy.Value} {s_skuLazy.Value} {s_runtimeVersionLazy.Value} {s_osLazy.Value} [{DateTime.UtcNow:MM/dd HH:mm:ss.ff}{_correlationId}]{ClientInformation} {messageToLog}";
+                string log = string.Format(
+                    CultureInfo.InvariantCulture,
+                    "{0} MSAL {1} {2} {3} {4} [{5}{6}]{7} {8}",
+                    isLoggingPii,
+                    s_msalVersionLazy.Value,
+                    s_skuLazy.Value,
+                    s_runtimeVersionLazy.Value,
+                    s_osLazy.Value,
+                    DateTime.UtcNow.ToString("u"),
```

Comment:
Date in universal format which LogsMIner uses.

Reply:
minor: Can this have string interpolation. That will make it easier to read the code with these many parameters.

Reply:
The reason I moved away from string interpolation to string.format is the culture. By default string interpolation uses current culture. This is not what we want and is a bit slower to process than invariant culture. 

There is a specify string interpolation's culture, but not in net45

https://stackoverflow.com/questions/33203261/what-is-the-default-culture-for-c-sharp-6-string-interpolation

Reply:
Note: we probably don't need to do this everywhere, but this method is perf critical.